guides/v2.2/frontend-dev-guide/layouts/xml-manage.md

```diff
@@ -383,7 +382,83 @@ In this file, reference the element having added the `remove` attribute:
+
+The following block, which is our old block, is declared in `app/design/frontend/catalog/product_view_attributes/layout/local.xml`:
+
+```xml
+<block type="catalog/product_view_attributes" name="product.attributes" as="additional" template="catalog/product/view/attributes.phtml">
+    <action method="addToParentGroup"><group>detailed_info</group></action>
+    <action method="setTitle" translate="value"><value>Additional Information</value></action>
+</block>
+```
+You can replace this block in `layout.xml` with a custom block that uses the same name, as follows:
+
+```xml
+<block type="attributesasgroup/groupview" name="product.attributes" as="additional" template="webguys/attributesasgroup/groupview.phtml">
+    <action method="addToParentGroup"><group>detailed_info</group></action>
+    <action method="setTitle" translate="value"><value>Additional Information</value></action>
+</block>
+```
+
+Magento processes the layout updates made within the special `local.xml` file last, after applying all other layout updates from other files. This means that you can include a `layout.xml` inside your theme's layout directory to ensure that your layout updates will replace an existing block type with the new block that defines your changes. For example:
+
+```xml
+<catalog_product_view>
+
+    <remove name="product.attributes" />
+
+    <reference name="content">
+        <block type="attributesasgroup/groupview" name="product.attributes" as="additional" template="webguys/attributesasgroup/groupview.phtml">
+            <action method="addToParentGroup"><group>detailed_info</group></action>
+            <action method="setTitle" translate="value"><value>Additional Information</value></action>
+        </block>
+    </reference>
+
+</catalog_product_view>
+```
+This layout structure held by our `local.xml` does two things. First, it removes the default block from the layout structure with the `remove` node.
+Second, it then adds our own custom block to the content section of the page within the `<reference>` container.
+
+As a final note, it is important that your block be defined on the backend side. If it is not defined, Magento will not understand which block you are referencing to it. This will mean that Magento will not properly render the content that is located inside your block.
+
+For instance, in order for the new block in the above case to be processed properly, Magento needs `Namespace_Modulename_Block_Groupview` to be defined at `app/code/local/Namespace/Modulename/Block/Groupview.php` (where `Namespace_Modulename` stands for your custom module name).
+
+Some block definitions do not contain an explicit name attribute, which means you cannot override the block from a layout XML file. For these blocks, you can use the class preference method to override the template.
+
+**Example 2**: override the block containing the `cart/item/default.phtml` template found in `Magento/Checkout/view/frontend/layout/checkout_cart_item_renderers.xml`.
```

Comment:
While this is technically possible, I don't think this is a good approach. Now, we can add functionality through [ViewModels](https://magento.stackexchange.com/questions/227725/magento-2-how-to-add-a-viewmodel-to-a-core-block) (this would be a good thing for a DevDocs topic) and we can modify functionality through [plugins (interceptors)](https://devdocs.magento.com/guides/v2.2/extension-dev-guide/plugins.html). This relegates overriding blocks to rare instances where its necessary to change something in a private/protected method, or something like that. There is a [topic that currently addresses the `<preference/>`](https://devdocs.magento.com/guides/v2.2/extension-dev-guide/build/di-xml-file.html#abstraction-implementation-mappings) and it can be used for any block.